tests/end2end/data/fileselect/fileselect.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Testing fileselection</title>
+        <script type="text/javascript">
+            function onLoad() {
+                for (let id of ['#single_file', '#multiple_files']) {
+                    const input = document.querySelector(id);
+                    input.addEventListener('input', function (e) {
```

Comment:
Looks like QtWebKit doesn't support [`input` events](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event) on `file` inputs - it does however seem to support [`change` events](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event), which for our purposes should be equivalent.

Thus, this should fix the QtWebKit failure (and still work on QtWebEngine):

```suggestion
                    input.addEventListener('change', function (e) {
```

Reply:
Changed